### PR TITLE
Switch on/off configuration for enabling comments/likes by default

### LINKED ISSUE #36: Allow interactivity to be turned on/off for images
Is there a way to make it so the below interactivity would be added to images within the content?

```
<figure data-feedback="fb:likes, fb:comments">
</figure>
```

### PR DESCRIPTION
**Pull request by everton-rosario**
This PR:

* [x] Adds configuration for likes and comments
* [x] Makes easier to enable likes + comments without any coding

Relates to https://github.com/facebook/facebook-instant-articles-sdk-php/pull/195

Fixes #36 and #340

### CODE REVIEW

File: class-instant-articles-post.php [hunk 1]
```diff
@@ -685,6 +685,15 @@ public function to_instant_article() {
 			$this->instant_article->withStyle( 'default' );
 		}
 
+		if ( isset( $settings_publishing[ 'likes_on_media' ] ) ) {
```
Comment (diegoquinteiro) [inline class-instant-articles-post.php]: Please add a comment to make it explicit that these settings affect the outcome of `$transformer->transformString(...)`

File: class-instant-articles-post.php [hunk 2]
```diff
@@ -685,6 +685,15 @@ public function to_instant_article() {
...
+			Image::$DEFAULT_LIKE_ENABLED = $settings_publishing[ 'likes_on_media' ];
```
Comment (diegoquinteiro) [inline class-instant-articles-post.php]: Please replace with the setters you'll implement in the SDK.

**Comment by diegoquinteiro** [comment]
@everton-rosario please resolve the conflicts so we can merge.

**Comment by everton-rosario** [comment]
@diegoquinteiro this should be good to go, once checks are finished.

**Comment by everton-rosario** [comment]
Closing this merged problems PR. 
Replaced by #617

**Pull request closed without merging**